src/coreclr/build.cmd

```diff
@@ -907,20 +898,6 @@ exit /b 1
 :ExitWithCode
 exit /b !__exitCode!
 
-:BuildCrossOSDac
-setlocal
-set __BuildDacOption=%1
-set __NextCmd=call %__ThisScriptFull% %__BuildDacOption% %__BuildArch% %__BuildType% %__PassThroughArgs%
-echo %__MsgPrefix%Invoking: %__NextCmd%
-%__NextCmd%
```

Comment:
Based on looking at the official builds that failed it seems this call doesn't generate a binary log file even though pretty much all of our other build commands do. How can we change this in the future to flow through the binary log file.